src/drivers/drv_sensor.h

```diff
@@ -108,7 +108,7 @@
 #define DRV_DIFF_PRESS_DEVTYPE_SDP33    0x4B
 #define DRV_BARO_DEVTYPE_LPS33HW        0x4C
 #define DRV_BARO_DEVTYPE_TCBP001TA      0x4D
-
+#define DRV_IMU_DEVTYPE_ICM42670P       0x4E
```

Comment:
Maybe move it up to 0x2A which is grouped with more IMUs including the DRV_IMU_DEVTYPE_ICM42605?